scripts/metamath-extract.py

```diff
@@ -175,10 +177,11 @@ def slice_database(input_database: Database) -> Iterator[tuple[str, Database]]:
             cut_antecedents[axiom_get_label(statement)] = cast('AxiomaticStatement | Block', statement)
         elif isinstance(statement, (ProvableStatement, Block)):
             antecedents, consequent = deconstruct_provable(statement)
-            yield (
-                consequent.label,
-                supporting_database_for_provable(cut_antecedents, global_disjoints, consequent, antecedents),
-            )
+            if (include and consequent.label in include) and (consequent.label not in exclude):
```

Comment:
Should this not be 
```
if (not include or consequent.label in include) and (consequent.label not in exclude):
```
? I would assume that having an empty `include` means that this option is not used, and thus everything is included.